open-media-match/.devcontainer/omm_config.py

```diff
@@ -0,0 +1,6 @@
+# Database configuration
```

Comment:
I'm inherently suspicious of turing complete configuration files, but I think this is fine in the short term. 

I've read enough about YAML to be scared, and tend towards just JSON, but this is all irrelevant in first pass.

Reply:
Fair point, and Flask provides a few methods by which to load config.